litespi/modules/modules.py

```diff
@@ -3,3 +3,52 @@
 from litespi.ids import SpiNorFlashManufacturerIDs
 
 # Define non-generated SPI NOR chips here
+
+class MX25L6406E(SpiNorFlashModule):
+
```

Comment:
Can you add a link to the datasheets in this module?

Reply:
This should be rather a triple quoted docstring or a comment above each class?

Reply:
I added them as comments above each class (direct links to pdfs).